index.js

```diff
@@ -339,16 +352,140 @@ class Raspi extends events.EventEmitter {
     }
   }
 
+  i2cConfig(delay) {
+    if (this[i2c] === undefined) {
```

Comment:
If I'm not mistaken, this isn't going to work the way things are currently implemented. `I2C` extends `Peripheral` and the `I2C` constructor calls the `Peripheral` constructor. The first thing the `Peripheral` constructor does is:
```JavaScript
this.alive = true;
```
`this[i2c]` would therefore always be alive so the checks and error throwing will not be possible.

Also, the I2C constructor calls `execSync('gpio load i2c')`. This should only happen if the user really wants to use I2C.

I think it would be better to leave thing the way they are here.